sample/core.py

```diff
@@ -80,6 +73,16 @@ def get_linkshere(self, lhprop="pageid|title|redirect", lhlimit="max"):
         self.payload['prop'] = None
         return lh_list
 
+    def _parse_kwargs(self, **kwparams):
+        if 'pageids' in kwparams:
+            self.payload['pageids'] = '|'.join(str(ID) for ID in list(kwparams['pageids']))
+        elif 'titles' in kwparams:
+            self.payload['titles'] = '|'.join(str(ID) for ID in list(kwparams['titles']))
+        elif 'revids' in kwparams:
+            self.payload['revids'] = '|'.join(str(ID) for ID in list(kwparams['revids']))
+        else:
+            raise KeyError("No valid arguments passed!")
```

Comment:
Sorry I forgot to mention this the first time. This one should be a [`ValueError`](https://docs.python.org/2/library/exceptions.html#exceptions.ValueError) which is preferred when the argument is invalid.